public/pages/Snapshots/components/RestoreActivitiesPanel/RestoreActivitiesPanel.tsx

```diff
@@ -12,14 +12,16 @@ import { getErrorMessage } from "../../../../utils/helpers";
 import { GetIndexRecoveryResponse, CatSnapshotIndex } from "../../../../../server/models/interfaces";
 import { BREADCRUMBS } from "../../../../utils/constants";
 import { ContentPanel } from "../../../../components/ContentPanel";
+import IndexList from "../IndexList";
 
 interface RestoreActivitiesPanelProps {
   snapshotManagementService: SnapshotManagementService;
   snapshotId: string;
```

Comment:
Only one snapshotId as the input, does this mean this panel can only show restore activities of one snapshot?